src/allmydata/test/test_system.py

```diff
@@ -629,7 +660,7 @@ def _render_config(config):
     """
     Convert a ``dict`` of ``dict`` of ``bytes`` to an ini-format string.
     """
-    return "\n\n".join(list(
+    return u"\n\n".join(list(
```

Comment:
Hmm, the docstring for this and the other .ini-generator helpers say they operate on bytes.  Maybe they operate on unicode strings now?